### ISSUE TITLE
Label: replace with an image ?

### ISSUE THREAD

**Issue by AnneHuSKa** [InseeFr/Lunatic#237]
For Business Outlook Surveys, most of the labels are replaced by arrows.
In DDI, we  have  for instance :
```xml
<l:Generation>
 <r:OtherMaterial>
  <r:Agency>fr.insee</r:Agency> <r:ID>INSEE-COMMUN-OM-hausse_stable_baisse-3</r:ID>
<r:Version>1</r:Version>
<r:TypeOfMaterial controlledVocabularyID="INSEE-TOM-CV">image</r:TypeOfMaterial>
<r:ExternalURLReference>stable.jpg</r:ExternalURLReference></r:OtherMaterial>
</l:Generation> 
```

What can be done in Lunatic ?

**Comment by romaintailhurat** [comment]
We are using https://github.com/remarkjs/react-markdown for parsing markdown in Lunatic. This lib is based on remark which enables the use of plugins.

So, we could use for example https://github.com/rhysd/remark-emoji to produce "images".

A emblematic use case is the use of ↗️ , ➡️ and ↘️ in forecast surveys, which would be supported out-of-the-box. 😄

**Comment by ddecrulle** [comment]
And if you want some other picture, use markdown syntaxe `![](links)`

**Comment by ddecrulle** [comment]
We have revert this feature due to a technical issue

**Comment by Grafikart** [comment]
It is fixed for v3.0
